deploy/templates/gcs-manifests/gcs-gd2-services.yml.j2

```diff
@@ -47,6 +47,8 @@ spec:
       port: 24007
       targetPort: 24007
 
```

Comment:
One last item... This whitespace goes below the metrics port.

Reply:
Ack